src/App.tsx

```diff
@@ -1,26 +1,25 @@
-import { Global, ThemeProvider } from '@emotion/react'
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import { ThemeProvider, Global } from '@emotion/react'
 import { resetStyle } from '@/styles/reset'
-import theme from './styles/theme'
+import theme from '@/styles/theme'
+import HomePage from '@/pages/HomePage'
+import LoginPage from '@/pages/LoginPage'
+import NotFoundPage from '@/pages/NotfoundPage'
 import { MainLayout } from './components/MainLayout'
-import Header from './components/Header'
-import Category from './components/Category'
-import Friends from './components/Friends'
-import Banner from './components/Banner'
-import TimeRanking from './components/TimeRanking'
 
 function App() {
   return (
     <ThemeProvider theme={theme}>
       <Global styles={resetStyle} />
-      <MainLayout>
-        <Header />
-        <Friends />
-        <Category />
-        <Banner />
-        <TimeRanking />
-      </MainLayout>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<MainLayout><HomePage /></MainLayout>} />
+          <Route path="/login" element={<MainLayout><LoginPage /></MainLayout>} />
+          <Route path="*" element={<MainLayout><NotFoundPage /></MainLayout>} />
+        </Routes>
```

Comment:
Route를 잘 적용해주셨군요 ㅎㅎ
여기서 반복되는게 MainLayout인데, 이렇게 표현할 수 있도록 컴포넌트를 하나 만들어주면 어떨까요?

```suggestion
        <Routes>
          <MainLayoutRoute path="/" element={<HomePage /> />
          <MainLayoutRoute  path="/login" element={<LoginPage />} />
          <MainLayoutRoute  path="*" element={<NotFoundPage />} />
        </Routes>
```

혹은 더 좋은 방법이 있는지 찾아보셔도 좋을 것 같아요!